_tests_/index.phtml

```diff
@@ -1,93 +1,17 @@
-<?php declare(strict_types=1);
+<?php
 
-use Magento\Framework\Escaper;
-use Magento\Framework\View\Element\Template;
+$greeting = "hello there";
 
-/** @var Escaper $escaper */
-/** @var Template $block */
 ?>
 
-<div>
-    <p class="greeting"><?= $escaper->escapeHtml(__('Hello there')) ?></p>
-</div>
+<p class="greeting"><?= $greeting; ?></p>
 
 <style>
-.order-example {
-    /* var's/special setters */
-    --color-theme: hsl(270 50% 40%);
-    content: attr();
-    counter-reset: section;
-    counter-increment: section;
-    quotes: "“" "”" "‘" "’";
-    box-sizing: border-box;
+  body {
+    margin: 0;
+  }
 
-    /* Positioning */
-    z-index: 1;
-    position: absolute;
-    right: 0;
-
-    /* Layout */
-    display: block;
-    float: left;
-    grid-column: span 2;
-    flex-flow: row wrap;
-    justify-content: center;
-    align-items: center;
-    table-layout: fixed;
-
-    /* Shape */
-    width: auto;
```

Comment:
Waarom zijn al deze dingen verwijderd? 🤔

Reply:
Omdat dit al getest wordt in de index.css, maakt ook de tests simpler om te beheren